### PR TITLE
Market data frankfurtmain

### PR DESCRIPTION
**Pull request by masterofpuppets103**
Added data of markets from Frankfurt am Main. Greetings from OKLab Frankfurt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ in February 2016.
 |[Düsseldorf][duesseldorf-wikipedia]|[City of Düsseldorf][duesseldorf-markets]|
 |[Erlangen][erlangen-wikipedia]|[City of Erlangen][erlangen-markets]|
 |[Essen][essen-wikipedia]|[EVB][essen-markets]|
+|[Frankfurt am Main][frankfurt-wikipedia]|[City of Frankfurt am Main][frankfurtmain-markets]|
```
Comment (johnjohndoe) [inline README.md]: The markdown link does not match the bottom link. You can always check the [preview of the `README.md` in you local branch](https://github.com/masterofpuppets103/wo-ist-markt.github.io/tree/market-data-frankfurtmain).

readme

File: cities/cities.json [hunk 2]
```diff
@@ -63,10 +63,14 @@
         "id": "essen",
         "label": "Essen"
     },
-    "freiburg": {
+    "frankfurtmain": {
```
Comment (johnjohndoe) [inline cities/cities.json]: You mixed up freiburg and frankfurtmain. Please fix.

File: cities/frankfurtmain.json [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+        }
+
+    ],
+    "metadata": {
+        "data_source": {
+            "title": "Stadt Frankfurt am Main",
+            "url": "http://frankfurt.de/sixcms/detail.php?id=4623&_ffmpar[_id_eltern]=4622"
+        },
+        "map_initialization": {
+            "coordinates": [
```
Comment (johnjohndoe) [inline cities/frankfurtmain.json]: The map initialization should be the center of the markets so it shows nice on a small screen. Here is how it looks currently:

smartphone

**Comment by johnjohndoe** [comment]
The citykey/id mismatch in cities.json should be automatically validated in the future, see #162. If you want to help with code: you are welcome!

**Comment by masterofpuppets103** [comment]
Hi,
thanks for the help. I did the changes you requested. (and I hope I commited it right. If not, just tell me)
Greetings

**Comment by masterofpuppets103** [comment]
Hi, thanks for the help. I did the changes you requested.

**Comment by johnjohndoe** [comment]
Thanks. Just for the note: somebody else has to work on this - I am on
vacation.

File: cities/frankfurtmain.json (hunk 3 above)
Comment (johnjohndoe) [inline cities/frankfurtmain.json]: The markets on the map are still not centered. I will adjust this before I merge your branch.

**Comment by johnjohndoe** [comment]
I squashed your commits into a single commit and merged the remaining branch here: eb9fec5219dd7cf8ee2059e3bbfcaef04ef8a546. This means:

Frankfurt am Main, welcome to _Wo ist Markt?_ :rocket: Thank you @masterofpuppets103 for your contribution :tulip: 

In order to promote the project I would like to ask you for two tasks:
- [ ] Send a tweet `#woistmarkt` with a **screenshot** and a **link** to the website (please share the tweet link here)
- [ ] Try to convince the city of Frankfurt to **link** to our website, see #38 

If you want to help with **code** take a look at our issue tracker.

**Pull request closed without merging**